<!DOCTYPE html><html lang="en-us" >

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.8.0">

  

  
  
  
  
  
  

  

  
  
  
    
  
  <meta name="description" content="major histocompatibility complex, class I, A
Gene Context Sentence Table 2. Analysis of context sentence of HLA-A gene in 21 abstracts.   PMID  Gene Context Sentence      32295479  After predicting and evaluating of the third structure of the protein candidate, the best 3 D predicted model was applied for docking studies with Toll-like receptor 4 (TLR4) and HLA-A11:01.">

  
  <link rel="alternate" hreflang="en-us" href="https://platcovid.com/genepanel/3105/gene_context/">

  


  
  
  
  <meta name="theme-color" content="#2962ff">
  

  
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css" integrity="sha256-4w9DunooKSr3MFXHXWyFER38WmPdm361bQS/2KUWZbU=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      
        
      

      
    
      

      
      

      
    

  

  
  
  
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap">
  

  
  
  
  
  <link rel="stylesheet" href="/css/academic.css">

  





<script async src="https://www.googletagmanager.com/gtag/js?id=UA-166873339-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];

  function gtag() {
      dataLayer.push(arguments);
  }

  function trackOutboundLink(url, target) {
    gtag('event', 'click', {
         'event_category': 'outbound',
         'event_label': url,
         'transport_type': 'beacon',
         'event_callback': function () {
           if (target !== '_blank') {
             document.location = url;
           }
         }
    });
    console.debug("Outbound link clicked: " + url);
  }

  function onClickCallback(event) {
    if ((event.target.tagName !== 'A') || (event.target.host === window.location.host)) {
      return;
    }
    trackOutboundLink(event.target, event.target.getAttribute('target'));  
  }

  gtag('js', new Date());
  gtag('config', 'UA-166873339-1', {});

  
  document.addEventListener('click', onClickCallback, false);
</script>


  


  
  

  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/images/icon_hu7e12209333f22d741aca7d462871821f_9877_32x32_fill_lanczos_center_2.png">
  <link rel="apple-touch-icon" type="image/png" href="/images/icon_hu7e12209333f22d741aca7d462871821f_9877_192x192_fill_lanczos_center_2.png">

  <link rel="canonical" href="https://platcovid.com/genepanel/3105/gene_context/">

  
  
  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="PlatCOVID">
  <meta property="og:url" content="https://platcovid.com/genepanel/3105/gene_context/">
  <meta property="og:title" content="HLA-A | PlatCOVID">
  <meta property="og:description" content="major histocompatibility complex, class I, A
Gene Context Sentence Table 2. Analysis of context sentence of HLA-A gene in 21 abstracts.   PMID  Gene Context Sentence      32295479  After predicting and evaluating of the third structure of the protein candidate, the best 3 D predicted model was applied for docking studies with Toll-like receptor 4 (TLR4) and HLA-A11:01."><meta property="og:image" content="https://platcovid.com/images/icon_hu7e12209333f22d741aca7d462871821f_9877_512x512_fill_lanczos_center_2.png">
  <meta property="twitter:image" content="https://platcovid.com/images/icon_hu7e12209333f22d741aca7d462871821f_9877_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us">
  
    
      <meta property="article:published_time" content="2019-05-05T00:00:00&#43;01:00">
    
    <meta property="article:modified_time" content="2019-05-05T00:00:00&#43;01:00">
  

  



  


  


  





  <title>HLA-A | PlatCOVID</title>

</head>

<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" >

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  







<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/">PlatCOVID</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/">PlatCOVID</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/syntax/"><span>Syntax</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link  active" href="/genepanel/"><span>Gene Panel</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/drugpanel/"><span>Drug Panel</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/tissuepanel/"><span>Tissue Panel</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/syntax_summary/"><span>Literature Consensus</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#about"><span>Contact</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      

      

    </ul>

  </div>
</nav>


  

<div class="container-fluid docs">
  <div class="row flex-xl-nowrap">
    <div class="col-12 col-md-3 col-xl-2 docs-sidebar">
      





  
    
  




<form class="docs-search d-flex align-items-center">
  <button class="btn docs-toggle d-md-none p-0 mr-3" type="button" data-toggle="collapse" data-target="#docs-nav" aria-controls="docs-nav" aria-expanded="false" aria-label="Toggle section navigation">
    <span><i class="fas fa-bars"></i></span>
  </button>

  
  <input name="q" type="search" class="form-control" placeholder="Search..." autocomplete="off">
  
</form>

<nav class="collapse docs-links" id="docs-nav">
  

  
  
  
  
  <div class="docs-toc-item">
    <a class="docs-toc-link" href="/genepanel/3105/"></a>

  </div>
  
  <div class="docs-toc-item">
    <a class="docs-toc-link" href="/genepanel/3105/info/">Overview</a>
    <ul class="nav docs-sidenav">
      
      <li >
        <a href="/genepanel/3105/gene_expression/">Gene Expression</a>
      </li>
      
      <li class="active">
        <a href="/genepanel/3105/gene_context/">Gene Context</a>
      </li>
      
      <li >
        <a href="/genepanel/3105/clinvar/">ClinVar</a>
      </li>
      
      <li >
        <a href="/genepanel/3105/geo/">Geo DataSet</a>
      </li>
      
    </ul>
    

  </div>
  
  
</nav>

    </div>

    

    <main class="col-12 col-md-9 col-xl-8 py-md-3 pl-md-5 docs-content" role="main">

      <article class="article">

        <div class="docs-article-container">
          <h1>HLA-A</h1>

          <div class="article-style">
            
<script src="/rmarkdown-libs/header-attrs/header-attrs.js"></script>
<script src="/rmarkdown-libs/kePrint/kePrint.js"></script>
<link href="/rmarkdown-libs/lightable/lightable.css" rel="stylesheet" />


<hr />
<p>major histocompatibility complex, class I, A</p>
<hr />
<div id="gene-context-sentence" class="section level2">
<h2>Gene Context Sentence</h2>
<br>
<p style="font-size:12px" align="center">
<strong>Table 2.</strong> Analysis of context sentence of HLA-A gene in 21 abstracts.
</p>
<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
PMID
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Gene Context Sentence
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32295479" target="_blank">32295479</a>
</td>
<td style="text-align:center;">
After predicting and evaluating of the third structure of the protein candidate, the best 3 D predicted model was applied for docking studies with Toll-like receptor 4 (TLR4) and HLA-A<em>11:01. […] In the next step, molecular dynamics (MD) simulation was used to evaluate the stability of the designed fusion protein with TLR4 and HLA-A</em>11:01 receptors. […] MD studies demonstrated that the NOM-TLR4 and NOM-HLA-A*11:01 docked models were stable during simulation time. <i>In silico</i> evaluation showed that the designed chimeric protein could simultaneously elicit humoral and cell-mediated immune responses.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32424945" target="_blank">32424945</a>
</td>
<td style="text-align:center;">
Here, 82 individuals with COVID-19 were genotyped for HLA-A, -B, -C, -DRB1, -DRB3/4/5, -DQA1, -DQB1, -DPA1, and -DPB1 loci using next-generation sequencing.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32475052" target="_blank">32475052</a>
</td>
<td style="text-align:center;">
Among the strong HLA binders (IC<sub>50</sub> ≤ 50), we uncovered 16 alleles (the top ones being A<em>02:02, B</em>15:03 and DRB1<em>01:02) binding more than 1% of peptides derived from all viruses, 9 (top ones including HLA-A</em>68:01, B<em>15:25, C</em>03:02 and DRB1<em>07:01) binding all viruses except HIV-1, and 15 (top ones A</em>02:01 and C*14:02) only binding coronaviruses.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32623831" target="_blank">32623831</a>
</td>
<td style="text-align:center;">
A bias towards an increased representation of HLA-A<em>26, HLA-DRB1</em>15, HLA-DRB1<em>10 and DRB1</em>11 was also noted but these were either only significant using the UK donor controls, or did not remain significant after correction for multiple tests. […] Likewise, HLA-A<em>02, HLA-B</em>44 and HLA-C*05 may exert a protective effect, but these associations did not remain significant after correction for multiple tests.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32699345" target="_blank">32699345</a>
</td>
<td style="text-align:center;">
We also found that the frequency of several HLA alleles, including HLA-A*11:01, were significantly associated with the fatality rates, although these factors were associated with number of infected cases and not an independent factor to affect fatality rate in each country.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32717807" target="_blank">32717807</a>
</td>
<td style="text-align:center;">
Among all the alleles, HLA-A<em>25, B</em>08, B<em>44, B</em>15:01, B<em>51, C</em>01, and C<em>03 showed a positive log-linear correlation with COVID-19 incidence rate fixed on 9 April 2020 in proximity of the national outbreak peak (Pearson’s coefficients between 0.50 and 0.70, p-value &lt; 0.0001), whereas HLA-B</em>14, B<em>18, and B</em>49 showed an inverse log-linear correlation (Pearson’s coefficients between -0.47 and -0.59, p-value &lt; 0.0001).
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32759312" target="_blank">32759312</a>
</td>
<td style="text-align:center;">
We identified 24 immunogenic epitopes in the SARS-CoV-2 protein S that could interact with 17 different MHC-I alleles (namely, HLA-A<em>01:01; HLA-A</em>02:01; HLA-A<em>11:01; HLA-A</em>24:02; HLA-A<em>68:01; HLA-A</em>23:01; HLA-A<em>26:01; HLA-A</em>30:02; HLA-A<em>31:01; HLA-B</em>07:02; HLA-B<em>51:01; HLA-B</em>35:01; HLA-B<em>44:02; HLA-B</em>35:03; HLA-C<em>05:01; HLA-C</em>07:01 and HLA-C*15:02) in the Brazilian population.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32913053" target="_blank">32913053</a>
</td>
<td style="text-align:center;">
Experience with influenza shows that infection primes CD8+ T cell memory to peptides presented by common HLA types like HLA-A2, which enhances recovery and diminishes clinical severity upon reinfection. […] We identified two HLA-A<em>02:01-restricted SARS-CoV-2-specfic CD8+ T cell epitopes, A2/S269-277 and A2/Orf1ab3183-3191 Using peptide-HLA tetramer enrichment, direct ex vivo assessment of A2/S269+CD8+ and A2/Orf1ab3183+CD8+ populations indicated that A2/S269+CD8+ T cells were detected at comparable frequencies (∼1.3 × 10-5) in acute and convalescent HLA-A</em>02:01+ patients. […] These frequencies were higher than those found in uninfected HLA-A*02:01+ donors (∼2.5 × 10-6), but low when compared to frequencies for influenza-specific (A2/M158) and Epstein-Barr virus (EBV)-specific (A2/BMLF1280) (∼1.38 × 10-4) populations.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32933522" target="_blank">32933522</a>
</td>
<td style="text-align:center;">
In this study we analysed the regional frequencies for the most common Italian haplotypes from the Italian Bone Marrow Donor Registry (HLA-A, -B, -C and -DRB1 at four-digit level). […] In this study we found that the two most frequent HLA haplotypes in the Italian population, HLA-A<em>:01:01g-B</em>08:01 g-C<em>07:01g-DRB1</em>03:01g and HLA-A<em>02.01g-B</em>18.01g-C<em>07.01g-DRB1</em>11.04g, had a regional distribution overlapping that of Covid-19 and showed respectively a positive (suggestive of susceptibility) and negative (suggestive of protection) significant correlation with both Covid-19 incidence and mortality.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/32988645" target="_blank">32988645</a>
</td>
<td style="text-align:center;">
We found a trend to a higher rate of the alleles HLA-A<em>32 (p=0.004) in healthy controls than in COVID-19 patients, and of the alleles HLA-B</em>39 (p=0.02) and HLA-C<em>16 (p=0.02) in COVID-19 patients than in healthy controls; however, all these p-values were not significant after correction for multiple comparisons. […] Logistic regression analysis showed that the presence of certain alleles was associated with higher mortality, such as the allele HLA-A</em>11 after controlling for SOFA (OR=7.693; 95% CI=1.063-55.650; p=0.04) or APACHE-II (OR=11.858; 95% CI=1.524-92.273; p=0.02), the allele HLA-C<em>01 after controlling for SOFA (OR=11.182; 95% CI=1.053-118.700; p=0.04) or APACHE-II (OR=17.604; 95% CI=1.629-190.211; p=0.02), and the allele HLA-DQB1</em>04 after controlling for SOFA (OR=9.963; 95% CI=1.235-80.358; p=0.03).
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33024578" target="_blank">33024578</a>
</td>
<td style="text-align:center;">
Finally, HLA-A*03 restricted 8-mer peptides are found to be shared broadly by human and coronaviridae helicases in functional hotspots, with potential implications for nucleic acid unwinding upon initial infection.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33047883" target="_blank">33047883</a>
</td>
<td style="text-align:center;">
We found that a possible association between HLA-A<em>02:01 and an increased risk for COVID-19. […] HLA-A</em>02:01 had a relatively lower capacity to present SARS-CoV-2 antigens compared with other frequent HLA class I molecules, HLA-A<em>11:01 or HLA-A</em>24:02. […] This study suggests that individuals with HLA-A<em>11:01 or HLA-A</em>24:02 genotypes may generate efficiently T-cell-mediated antiviral responses to SARS-CoV-2 compared with HLA-A*02:01.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33085221" target="_blank">33085221</a>
</td>
<td style="text-align:center;">
We characterized medication orders, focusing on medications with actionable PGx guidance related to 14 commonly assayed genes (CYP2C19, CYP2C9, CYP2D6, CYP3A5, DPYD, G6PD, HLA-A, HLA-B, IFNL3, NUDT15, SLCO1B1, TPMT, UGT1A1, and VKORC1).
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33268522" target="_blank">33268522</a>
</td>
<td style="text-align:center;">
For the development of vaccines based on SARS-CoV-2-specific cytotoxic T lymphocytes (CTLs), we attempted to identify HLA-A<em>02:01-restricted CTL epitopes derived from the non-structural polyprotein 1a of SARS-CoV-2. […] Out of 82 peptides predicted on bioinformatics, 54 peptides showed good binding affinities to HLA-A</em>02:01. […] Using HLA-A*02:01 transgenic mice, 18 in 54 peptides were found to be CTL epitopes in the intracellular cytokine staining assay.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33298875" target="_blank">33298875</a>
</td>
<td style="text-align:center;">
We identified that the HLA-A<em>11:01, B</em>51:01, and C*14:02 alleles significantly predispose the worst outcome of the patients.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33301503" target="_blank">33301503</a>
</td>
<td style="text-align:center;">
We found diverse capacities of S protein specific epitope presentation by different HLA alleles with very limited number of predicted epitopes for HLA-B<em>2705, HLA-B</em>4402 and HLA-B<em>4403 and as high as 132 epitopes for HLA-A</em>6601.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33325143" target="_blank">33325143</a>
</td>
<td style="text-align:center;">
Besides, docking analysis showed that the predicted peptide ‘KIADYNYKL’ was closely bound to the HLA-A<em>0201. […] The results of molecular dynamics simulation through GROMACS software showed that ’HLA-A</em>0201~peptide’ complex was very stable.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33343579" target="_blank">33343579</a>
</td>
<td style="text-align:center;">
The extended haplotype HLA-A<em>02:05, B</em>58:01, C<em>07:01, DRB1</em>03:01 [OR 0.1 (95% CI 0-0.6), Pc = 0.015] was absent in all 182 patients, while the HLA-C<em>04:01 allele and the three-loci haplotype HLA-A</em>30:02, B<em>14:02, C</em>08:02 [OR 3.8 (95% CI 1.8-8.1), Pc = 0.025] were more frequently represented in patients than controls. […] The data emerging from our study suggest that the extended haplotype HLA-A<em>02:05, B</em>58:01, C<em>07:01, DRB1</em>03:01 has a protective effect against SARS-CoV-2 infection in the Sardinian population.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33361777" target="_blank">33361777</a>
</td>
<td style="text-align:center;">
To help achieve these aims, we profiled the entire SARS-CoV-2 proteome across the most frequent 100 HLA-A, HLA-B and HLA-DR alleles in the human population, using host-infected cell surface antigen presentation and immunogenicity predictors from the NEC Immune Profiler suite of tools, and generated comprehensive epitope maps.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33403186" target="_blank">33403186</a>
</td>
<td style="text-align:center;">
Methods To clarify the relationship between HLA class I and the severity rate, the binding repertoires of each HLA class I allele to SARS-CoV-2 peptides and the allele frequencies of HLA-A, -B, and -A/B haplotypes in each country were quoted. […] Results HLA-A1 and the number of deaths per million population (severity rate) in each country had an exponential approximation correlation with correlation coefficient R=0.4879. […] Weak HLA-A alleles with a repertoire of under 300 also had an exponential approximation correlation with the severity rate (R=0.5972), whereas there was a linear approximation with the infection rate (R=0.6808). […] The weak HLA-A/B haplotype has a stronger effect on the severity rate than the weak HLA-A alone. […] The weak HLA-A is a major factor of severity rate, whereas the weak -B alone has no correlation. […] However, the weak HLA-A/B haplotype has a stronger effect on the severity rate than the weak -A alone.
</td>
</tr>
<tr>
<td style="text-align:center;">
<a href="https://pubmed.ncbi.nlm.nih.gov/33586620" target="_blank">33586620</a>
</td>
<td style="text-align:center;">
Secondly, the 3D model of selected epitopes and HLA-A<em>29:02 were built and Molecular Docking simulation was performed. […] Upon Molecular Docking, interactions between selected epitopes and HLA-A</em>29:02 were further validated by 50 ns Molecular Dynamics (MD) simulation.
</td>
</tr>
</tbody>
</table>
<hr />
</div>

          </div>

          



          
        </div>

        <div class="body-footer">
          
          

          
        </div>

      </article>

      <footer class="site-footer">
  
  <p class="powered-by">
    
      
      <a href="/supp/faq/">FAQ</a>
    
  </p>
  

  <p class="powered-by">
    BioHub - UFPE &middot; 

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
  </p>
</footer>


    </main>
  </div>
</div>


      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js" integrity="sha256-eOgo0OtLL4cdq7RdwRUiGKLX9XsIJ7nGhWEKbohmVAQ=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/languages/r.min.js"></script>
        
      

    

    
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    <script>const isSiteThemeDark = false;</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.1.1/anchor.min.js" integrity="sha256-pB/deHc9CGfFpJRjC43imB29Rse8tak+5eXqntO94ck=" crossorigin="anonymous"></script>
    <script>
      anchors.add();
    </script>
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    
    <script id="dsq-count-scr" src="https://procovid19.disqus.com/count.js" async></script>
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.4aaf2d64e50f8e54cf66dcc54147a22e.js"></script>

    






  
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
